exampleplugins/5.2/SwagAttribute/SwagAttribute.php

```diff
@@ -42,13 +42,12 @@ public function install(InstallContext $context)
             'custom' => true,
 
             //in case of combo box type, defines the selectable values
-            'arrayStore' => [
+            'store' => [
```

Comment:
are you sure this works? the column in s_attribute_configuration is called array_store